style.css

```diff
@@ -0,0 +1,106 @@
+header {
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  padding: 120px, 24px, 120px, 24px;
+  background-color: black;
+}
+
+.nav-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #3a3c39;
+  height: 92px;
+  width: 375px;
+  left: 0;
+  top: 0;
+}
+
+.time-network {
+  display: flex;
+  justify-content: space-between;
+  color: white;
+}
+
+.network-only {
+  display: flex;
+  list-style-type: none;
+  padding-right: 21px;
+}
+
+.network-only-item {
+  padding: 2px;
+}
+
+h4 {
+  height: 16px;
+  width: 54px;
+  left: 0;
+  top: 3px;
+  padding-left: 25px;
+}
+
+.name-and-burger {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3a3c39;
+}
+
+h2 {
+  color: white;
+  padding-left: 18px;
+}
+
+.burger {
+  padding-right: 29px;
+}
+
+.home-container {
+  display: flex;
+  padding-top: 120px;
+  background-color: black;
+  flex-direction: column;
+  padding-left: 24px;
+  padding-right: 24px;
+  background-image: url(image/Group.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom 13%;
+  height: 590px;
+}
```

Comment:
- [ ] Your mobile version of the app is not currently optimized well. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp). Please work on it. :writing_hand: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![m1](https://user-images.githubusercontent.com/70483835/164491329-29a34e84-756a-4cca-a487-ac80b7999271.png) | ![m2](https://user-images.githubusercontent.com/70483835/164491373-d9500d21-0301-4ec9-ab66-e6f558c1153c.png)